src/images/pages/regulation/map.svg

```diff
@@ -1,3565 +1 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
wait, if we're deleting the map.svg, how will we display the regulation map image?

Reply:
map svg should be updated, not deleted